salesforce-marketing-cloud/3.1/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,417 @@
+= Salesforce Marketing Cloud Connector 3.1 - Mule 4
+:page-aliases: connectors::salesforce/salesforce-mktg-connector.adoc
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
+
+Anypoint Connector for Salesforce Marketing Cloud (Marketing Cloud Connector) enables you to connect to the Marketing Cloud API web services (now known as the Marketing Cloud API), which is also known as the Salesforce Marketing Cloud. This connector exposes convenient operations for exploiting the capabilities of Salesforce Marketing Cloud.
+
+Release Notes: xref:release-notes::connector/salesforce-mktg-connector-release-notes-mule-4.adoc[Salesforce Marketing Cloud Connector Release Notes] +
+Exchange: https://www.mulesoft.com/exchange/com.mulesoft.connectors/mule-sfdc-marketing-cloud-connector/[Salesforce Marketing Cloud Connector]
+
+== Prerequisites
+
+To use this information, you should be familiar with Salesforce Marketing, Mule runtime engine (Mule), Anypoint Connectors, Anypoint Studio, Mule concepts, elements in a Mule flow, and Global Elements.
+
+You need login credentials to test your connection to your target resource.
+
+For software requirements and compatibility
+information, see the Connector Release Notes.
+
+== POM File Information
+
+[source,xml,linenums]
+----
+<dependency>
+  <groupId>com.mulesoft.connectors</groupId>
+  <artifactId>mule-sfdc-marketing-cloud-connector</artifactId>
+  <version>RELEASE</version>
+  <classifier>mule-plugin</classifier>
+</dependency>
+----
+
+Mule converts RELEASE to the latest version. To specify a version, view
```

Comment:
Let's update this to what we have in the template now and change RELEASE to x.x.x